docs/en/revocation-lists.rst

```diff
@@ -1,57 +1,520 @@
+- be non-repudiable even beyond its expiration time and even in the case of cryptographic keys rotation.
+
+
+**The Issuer MUST:**
+
+- ensure that the data contained in a Digital Credential is kept up to date, including the status of validity of the data from the Authentic Source;
+- revoke a Digital Credential when 
+
+  - it requires to be updated, whenever one or more attributes are changed; in this case the User will request a new issuance for that Digital Credential;
+  - the Wallet Instance to which it was issued is revoked;
+  - the User deletes it from the Wallet Instance;
+
+- provide a web service for allowing a Wallet Instance, with a proof of possession of a specific Digital Credential, to 
+
+  - request a revocation of that Digital Credential;
+  - obtain a related Status Attestation;
+
+- provide out-of-band mechanisms through which the User can request the revocation of their Digital Credentials, using a robust procedure for identity proofing and User authentication, in particular when the User is unable to use the personal Wallet Instance. 
+
+
+**The Wallet Instance MUST:**
+
+- check periodically the validity status of the Digital Credential that is stored in it, requesting a Status Attestation for each Digital Credential;
+- be able to present a Status Attestation if required by a Verifier, along with the corresponding Digital Credential;
+- request a revocation of a Digital Credential when the Users delete it from the storage. 
+
+
+**The Authentic Sources MUST:**
+
+- provide web services for the providing of updated User data and the validity status;
+- store in local databases only the minimum information required to provide the Issuer with the User data or a change in the validity status.
+
+
+Revocation Use Cases
+--------------------
+
+The revocation of a Digital Credential MAY be triggered by: 
+
+- Users using their personal Wallet Instance or by some out-of-band touchpoints.
+- Revocation of the Wallet Instance
+- Authentic Sources (e.g., for attribute updates) following administrative purposes. 
+- Law-Enforcing Bodies for the fulfillment of their functions and any other judicial reasons (e.g., Police).
+
+Credential Revocation Flows can start under different scenarios, such as:
+
+    - The User reports the loss or theft of their own physical document to the Law-Enforcement Authorities: this implies that the Credentials, if any, shall be revoked.
+    - The User notifies an Authentic Source that one or more attributes are changed (e.g. the current resident address): in this case the Credentials SHALL be revoked, as they are no longer valid due to the change in attributes. 
+    - The Users who no longer have access to their Wallet Instance (e.g. theft or loss of the device), may request the Issuer of the Credentials to revoke them or may request the Wallet Provider to revoke the Wallet Instance which in turn implies the revocation of all Digital Credentials stored in the Wallet Instance.
```

Comment:
```suggestion
    - Users who lose access to their Wallet Instance (e.g., due to theft or loss of the device) can request the Credential Issuer to revoke their Credentials or ask the Wallet Provider to revoke the Wallet Instance. If the Wallet Provider is authorized by the User and is aware of the types of Credentials and their issuers stored in the Wallet, it can then initiate the revocation of all Digital Credentials contained within the Wallet Instance on behalf of the User.
```